src/shared/sol-util-contiki.c

```diff
@@ -54,3 +54,10 @@ sol_util_timespec_get_realtime(struct timespec *t)
     errno = ENOSYS;
     return -1;
 }
+
+int
+sol_util_get_os_version(char **out)
+{
+    SOL_WRN("Not implemented");
```

Comment:
really? @ibriano and @zehortigoza do you know of some way to get OS version there? Maybe some macro?